server/controllers/recommendations/createAndDownloadDocument.ts

```diff
@@ -61,7 +61,6 @@ export const createAndDownloadDocument =
 
         const isSpoRecordedRationale = statuses.find(status => status.name === STATUSES.SPO_RECORDED_RATIONALE)
         if (!isSpo && isSpoRecordedRationale) {
-          activate.push(STATUSES.CLOSED)
```

Comment:
Have we missed something here?  We would have set the closed state for PART As and DNTRs before this change, but we now set SENT_TO_PPCS just for Part As.  Is that right?  DNTRs aren't sent to PPCS, but is there something we still need to do here for DNTRs?

Reply:
Yes DNTRs are now set to REC_CLOSED, see line 77